app/templates/views/send-one-off-letter-address.html

```diff
@@ -0,0 +1,45 @@
+{% extends "withnav_template.html" %}
+{% from "components/page-header.html" import page_header %}
+{% from "components/page-footer.html" import page_footer %}
+{% from "components/message-count-label.html" import recipient_count_label %}
+{% from "components/textbox.html" import textbox %}
+{% from "components/form.html" import form_wrapper %}
+
+{% block service_page_title %}
+  {{ page_title }}
+{% endblock %}
+
+{% block maincolumn_content %}
+
+  {{ page_header(
+    page_title,
+    back_link=back_link
+  ) }}
+
+  {% call form_wrapper(
+    class='js-stick-at-top-when-scrolling send-one-off-form',
```

Comment:
```suggestion
    class='send-one-off-form',
```
We probably don’t want the sticky behaviour here because scrolling is likely to obscure the address on the letter, which is the thing you’re trying to match up with the textbox. Still makes sense for later placeholders, which will be further down the page.